test/integration/dialects/postgres/query-interface.test.js

```diff
@@ -18,6 +18,68 @@ if (dialect.match(/^postgres/)) {
       this.queryInterface = this.sequelize.getQueryInterface();
     });
 
+    describe('createSchema', function () {
+      beforeEach( function () {
+        //make sure we don't have a pre-existing schema called testSchema.
+        return this.queryInterface.dropSchema('testschema')
+        .catch( err => {});
+      });
+
+      it('creates a schema', function () {
+        return this.queryInterface.createSchema('testschema')
+        .then( () => {
+          return this.sequelize.query('SELECT schema_name FROM information_schema.schemata WHERE schema_name = \'testschema\';', { type: this.sequelize.QueryTypes.SELECT });
+        })
+        .then ( res => {
+          expect(res, 'query results').to.not.be.empty;
+          expect(res[0].schema_name).to.be.equal('testschema');
+        });
+      });
+
+      it('errors if the schema exists', function () {
+        return this.queryInterface.createSchema('testschema')
+        .catch( err => {
+          return expect(err.toString()).to.be.equal('SequelizeDatabaseError: schema "testschema" already exists');
+        });
+      });
+    });
+
+    describe('databaseVersion', function () {
+      it('reports version', function () {
+        return this.queryInterface.databaseVersion()
+        .then( res => {
+          //check that result matches expected version number format. example 9.5.4
+          return expect(res).to.match(/[0-9\.[0-9]\.[0-9]/);
+        });
+      });
+    });
+
+    describe('renameFunction', function () {
+      beforeEach( function () {
+        //ensure the function names we'll use don't exist before we start.
+        //then setup our function to rename
+        return this.queryInterface.dropFunction('rftest1', [])
+        .catch( err => {})
+        .then( () => {
+          return this.queryInterface.dropFunction('rftest2', []);
+        })
+        .catch( err => {})
```

Comment:
please follow the general style of this repoistory (no space before the function parens/args for arrow methods)